policies/glossary.md

```diff
@@ -0,0 +1,151 @@
+# Glossary of OpenSSL terms
+
+## ABI
+
+Application binary interface.
+
+An ABI compatible release allows applications to
+be relinked against the new library with the expectation that everything will
```

Comment:
run? If it's a shared object, you can just update the library without relinking it.